src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -242,6 +236,15 @@ public void OnStopActivity(Activity activity, object payload)
 
                 activity.SetTag(SemanticConventions.AttributeHttpStatusCode, response.StatusCode);
 
+#if NET6_0_OR_GREATER
```

Comment:
could we reflection and fetch these in non .NET 6.0 scenarios?

Reply:
This is not available for < `net6.0` scenarios.